README.md

```
@@ -1,61 +1,409 @@
+
+3. **Environment Configuration**
+   - Duplicate `.env.example` and rename it to `.env`
+   - Fill in all required environment variables
+   - Configure Discord OAuth2 settings in your Discord Developer Portal
+
+4. **Database Setup**
+   - Ensure your WolfStar bot database is accessible
+   - Run database migrations if needed
+
+5. **Start the development server**
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)